mne/io/kit/constants.py

```diff
@@ -48,6 +48,7 @@
     50: (2, 1, 1),  # Hanger Type #3
     60: (2, 1, 1),  # Hanger Type #3
     100: (3, 3, 3),  # Low Band Kapper Type
+    101: (3, 3, 3),  # Low Band Kapper Type
```

Comment:
@UrbanM any idea which type you have? Or equivalently, does your acquisition/manufacturer software tell you somewhere what you highpass/lowpass/notch filter settings were during acquisition?

Reply:
I contacted the manager of the device, I will give you the answer soon.

Reply:
There is no default highpass/lowpass/notch filter settings, for this exact dataset it is HPF: TrueDC, LPF200Hz, BEF:Through . And for regular SQUID measurements we mostly use HPF: 0.1, LPF200Hz, BEF:Through. I hope this makes sense.